spec/src/main/asciidoc/opentelemetry-apis.adoc

```diff
@@ -0,0 +1,90 @@
+//
+// Copyright (c) 2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+[[sec:opentelemetry-api]]
+== Supported OpenTelemetry API Classes
+
+Classes from the following API packages MUST be supported by implementations of this specification, though this specification does not prevent additional API classes from being supported.
```

Comment:
The spec should define exactly what it means by "support" here (and, really, anywhere it uses that term).

For example, from this alone as written how would a TCK test author know how to write a test to verify that an implementation "supports" the classes in the packages listed? How would a vendor know how to implement such "support?" What behavior should an end user expect from a vendor's implementation of such "support?"

Reply:
Should we update this to say `Classes from the following API packages MUST be made available to the end users, though this specification does not prevent additional API classes from being exposed`?

Reply:
Yes, I've got `available to applications by implementations of this specification`.

Other uses of support throughout the spec are IMO clear.

Reply:
Just so we are clear... By "be made available" does the spec intend to convey the following:

A vendor's Maven artifact(s) will have dependencies on the Otel artifacts which contain the listed packages. Therefore, those Otel artifacts will be transitive dependencies for an application with a dependency on the vendor's artifact. This relieves the application developer of declaring explicit dependencies on the Otel artifacts.

I'm not necessarily suggesting that this wording be added to the spec. I just want to make sure I (and other readers) have a clear understanding what the spec requires.

Reply:
The requirement is weaker than that, as in current wording it doesn't prescribe the whole artifacts to be available. It guarantees that these packages and any of their dependents (which not many as those are only the APIs) does not need to be present in application artifact in order to successfully resolve them.

As to whether vendors would use Maven dependency or something else (AFAIK openliberty has a fork of those APIs in their repo) is not relevant. But for user it means, that relevant artifacts may be handles as `provided` (as long the application only touches the packages mentioned in the spec). Vendors can obviously place higher guarantees in their implementation.

Reply:
Good clarification.

As I said, I was not arguing for the spec to include the specific Maven phrasing I used; I meant to suggest only that the spec could more clearly define what "be made available" means. 

Maybe I'm being overly concerned about the wording here, but a spec is best when users and TCK test authors know from the text _exactly_ what behavior is prescribed and can be tested. The original use of "support" did not do that for me, and if "be made available" means "be on the compile-time and runtime classpaths" then can the spec just say that?